yt/geometry/coordinates/cartesian_coordinates.py

```diff
@@ -72,28 +137,28 @@ def _sample_ray(ray, npoints, field):
     return x, field_values
 
 def all_data(data, ptype, fields, kdtree=False):
-        field_data = {}
-        fields = set(fields)
+    field_data = {}
+    fields = set(fields)
+    for field in fields:
+        field_data[field] = []
+
+    for chunk in data.all_data().chunks([], "io"):
         for field in fields:
-            field_data[field] = []
+            field_data[field].append(chunk[ptype,
+                                           field].in_base("code"))
 
-        for chunk in data.all_data().chunks([], "io"):
-            for field in fields:
-                field_data[field].append(chunk[ptype,
-                                               field].in_base("code"))
+    for field in fields:
+        field_data[field] = uconcatenate(field_data[field])
 
+    if kdtree is True:
```

Comment:
Why does this drop out of the chunk loop on line 145

Reply:
Unfortunately the diff is not doing this justice. What happened in this `all_data` function is that it was indented 8 spaces to the right instead of 4 at the first level. I just moved it back. I'd check [this](https://github.com/yt-project/yt/blob/d281976893517adab229fd5c5856f256abf16b3e/yt/geometry/coordinates/cartesian_coordinates.py#L139-L161) instead of the diff to see what happened.

Reply:
Yeah, looks like this was missed in code review from one of Ash's GSOC pull requests.